embed/trezorhal/stm32_system.c

```diff
@@ -170,7 +170,7 @@ const uint32_t MSIRangeTable[12] = {100000, 200000, 400000, 800000, 1000000, 200
                is no need to call the 2 first functions listed above, since SystemCoreClock
                variable is updated automatically.
   */
-  uint32_t SystemCoreClock = 16000000;
```

Comment:
Yes, because the HSI is 16MHz and their code assumed that it would run while the HSI was still the clock source. I switched to 168MHz here because by the time that variable gets initialized for the C code, the system clock has already been switched over to the PLL driven by the HSE. I kept this variable around so that any CMSIS code that depended on it would not break.